Mod2/Lessons/Week2/GitForPairs.md

```diff
@@ -0,0 +1,132 @@
+
+> 4. P2: Accept GitHub invitation
+> 1. P2: Create new branch
+
+| Player 1 | Player 2 |
+| -------- | -------- |
+| | Accept invitation on GitHub<br>Clone `US_States` repo to local
+| | Checkout new branch: `colorado`<br>Add `colorado.txt` file<br>Add "Colorado" to `all_states.txt` file<br>`git add` , `git commit` , `git push`
+
+### Cloning a repository
+
+Player 2 has performed two important tasks. First, they have cloned the repo created by Player 1. To clone a repository means to copy a remote Github repository to your local computer using the following command:
+
+```
+git clone <SSH KEY for repository>
+```
+
+When you clone a repository, you interact with that repository by pushing and pulling branches. Therefore, when you are working with teammates, i.e. in a group project, you all need to clone the same repository so that you are all working in the same code base.
+
+### Pull Request
+
+Second, Player 2 has created content in a separate branch, but it not yet merged with the `main` branch. We've discussed how to merge branches, but we want one additional step to ensure that both developers agree on the content.
+
+To that end, Player 2 will create a Pull Request (PR). A **Pull Request** informs all collaborators of the changes made to a branch and the desire to merge this branch with another branch (usually the `main` branch).
+
+> 6. P2: Create Pull Request on GitHub
+
+| Player 1 | Player 2 |
+| -------- | -------- |
+| Review and comment on Player 2's PR<br>Merge Pull Request | |
+
+Player 1 can review, comment, and merge the Pull Request. Player 2 can also add commits to the PR. There can be several rounds of reviews, comments, and additional commits before it is merged.
+
+> 7. P1: Fix Merge Conflict
+
+| Player 1 | Player 2 |
+| -------- | -------- |
+| Still on `north_carolina` branch<br>`git pull origin main`<br>Fix Merge Conflict<br>`git add` , `git commit` , `git push` | |
+| Create Pull Request on GitHub | |
+
+There's **Good News** and _Bad News_. The **Good News** is that Player 2's content has been successfully merged into the shared repo. The _Bad News_ is that when we integrate those changes into our current branch, we encounter a pesky Merge Conflict. But there's more **Good News**: fixing a Merge Conflict when working with a collaborator is no more difficult than working solo.
+
+Once Player 1 fixes the Merge Conflict, they will update their branch and author a Pull Request of their own.
+
+### Merge and Update Main Branch
+
+> P2: Merge PR and Update `main` branch
+
+| Player 1 | Player 2 |
+| -------- | -------- |
+| | Review and comment on Player 1's PR<br>Merge Pull Request
+
+Let's answer a question before it's asked: **YES**, even though Player 1 owns this repo, they also make pull requests so that their changes are reviewed by the other collaborators. And **YES**, the process for Player 2 is the same.
```

Comment:
Love this callout that the 'owner' of the repo has no special powers :)

Reply:
I'll even add that point explicitly.